src/expr/src/scalar/func.rs

```diff
@@ -4066,6 +4066,17 @@ pub fn hmac_inner<'a>(
     Ok(Datum::Bytes(temp_storage.push_bytes(bytes)))
 }
 
+fn repeat_string<'a>(
+    datums: &[Datum<'a>],
+    temp_storage: &'a RowArena,
+) -> Result<Datum<'a>, EvalError> {
+    Ok(Datum::String(temp_storage.push_string(
+        datums[0].unwrap_str().repeat(
+            usize::try_from(datums[1].unwrap_int32()).map_err(|_| EvalError::Int32OutOfRange)?,
```

Comment:
`repeat('a', -1)` actually seems to return `""` rather than an error in Postgres. Go figure. 🤷🏽

Reply:
Hahaha, oops! I'll make it so